vortex-api/src/main/java/mil/army/usace/hec/vortex/io/NetcdfGridWriter.java

```diff
@@ -253,6 +254,10 @@ private void addVariableProjection(NetcdfFormatWriter.Builder writerBuilder) {
                     Attribute.builder().setName(name).setValues(Array.makeFromJavaArray(numericValues)).build();
             variableBuilder.addAttribute(attribute);
         }
+
+        // Adding CRS WKT for Grid's Coordinate System information
+        // CF Conventions: https://cfconventions.org/Data/cf-conventions/cf-conventions-1.11/cf-conventions.html#use-of-the-crs-well-known-text-format
+        variableBuilder.addAttribute(new Attribute(CRS_WKT, defaultCollection.getWkt()));
```

Comment:
Storing CRS WKT when writing out to NetCDF file -- ensuring more accurate WKT data (compared to using only Projection)

Reply:
I like adding this attribute. I'd kind of like to see the case that broke things so that I understand it better.